### PR TITLE
Issue38

### LINKED ISSUE #48: Rearrange sections
In the europe paper  : 

Methodology should include all of the input (historic deployments, predicted european growth, predicted decommissioning, material definitions, reactor details.)

Results should include all of the things that Cyclus and Cycamore produced in the output file and (SFR transition deployment ... which was found through the logic of the Cycamore GrowthRegion.)

Currently, these things are all mixed together. For example:

- All of section 6 should actually be part of section 3.2. 
- Section 3.3 (SFR deployment) should actually be part of results.

I have a pretty solid idea of how I want this to be ordered, so I'm going to give the rearrangement a shot. It will be an ugly PR... brace yourself @jbae11 .

### PR DESCRIPTION
**Pull request by katyhuff**
This adds a few paragraphs on the Polimi folks as well as the Delft folks. This closes #48.

### CODE REVIEW

File: Moltres.tex [hunk 1]
```diff
@@ -105,38 +105,52 @@ \section{Introduction}
...
+which this Politecnico di Milano approach was expanded to a multi-channel model 
+of the \gls{MSFR} and compared to code from the University of Delft
+\cite{de_zwaan_static_2007,van_der_linden_2012} based on the approach in
+\cite{kophazi_development_2009}. These models showed good agreement for
+multiple accident transients. Meanwhile, leveraging lessons learned from these 
+efforts has resulted in a multiscale approach from Zanetti et al. 
+\cite{zanetti_geometric_2015} successfully combines high and low geometric 
+fidelity for graphite-moderated \glspl{MSR}.
+
+More recently, Aufiero et.  al \cite{aufiero_development_2014} have begun to
```
Comment (andrewryh) [inline Moltres.tex]: Aufiero et.  al**_._**

File: Moltres.tex [hunk 2]
```diff
@@ -105,38 +105,52 @@ \section{Introduction}
...
+velocity field, used heterogeneous group constants in fuel and moderator
+regions, and employed the \gls{COMSOL} software package intrinsically designed
+for coupled multi-physics simulation.  Fiorina, Lathouwers, and their
+colleagues conducted a benchmarking exercise \cite{fiorina_modelling_2013} in
+which this Politecnico di Milano approach was expanded to a multi-channel model 
+of the \gls{MSFR} and compared to code from the University of Delft
+\cite{de_zwaan_static_2007,van_der_linden_2012} based on the approach in
+\cite{kophazi_development_2009}. These models showed good agreement for
+multiple accident transients. Meanwhile, leveraging lessons learned from these 
+efforts has resulted in a multiscale approach from Zanetti et al.
```
Comment (andrewryh) [inline Moltres.tex]: Zanetti et**.** al.

File: Moltres.tex [hunk 3]
```diff
@@ -105,38 +105,52 @@ \section{Introduction}
...
+efforts has resulted in a multiscale approach from Zanetti et al. 
+\cite{zanetti_geometric_2015} successfully combines high and low geometric 
+fidelity for graphite-moderated \glspl{MSR}.
+
+More recently, Aufiero et.  al \cite{aufiero_development_2014} have begun to
+approach transient simulations in the \gls{MSFR} within the finite volume
+OpenFOAM multiphysics toolkit\cite{weller_tensorial_1998}.  This approach
+benefits from pre-implemented turbulence models available in the OpenFOAM
+library and captures the full-core three-dimensional geometry of the reactor
+primary circuit.  OpenFOAM \gls{CFD} has additionally been shown by Laureau et
```
Comment (andrewryh) [inline Moltres.tex]: Laureau et**.** al.

**Review by andrewryh — APPROVED** [review]
I only changed "et al." 3 times, nice job!

**Comment by katyhuff** [comment]
thanks @andrewryh !

**Comment by katyhuff** [comment]
Given @andrewryh is ok with this, I'll merge it in so that we can keep plugging along.

**Pull request merged**